practice/pig-latin.py

```diff
@@ -1,58 +1,180 @@
-        re_capitlaize_word = False
-    elif english_word[0] in CAPITALS_LIST:
-        modified_english_word = english_word.lower()
-        re_capitlaize_word = True
-    else:
-        modified_english_word = english_word
-        re_capitlaize_word = False
-
-    if english_word[0] in FIRST_VOWEL:
-        pig_latin_word = modified_english_word + 'yay'
-    elif english_word[0] not in FIRST_VOWEL and english_word[1] in LATER_VOWEL:
-        pig_latin_word = modified_english_word[1:] + modified_english_word[:1] + 'ay'
-    elif english_word[0] not in FIRST_VOWEL and  english_word[1] not in LATER_VOWEL:
-        pig_latin_word = modified_english_word[2:] + modified_english_word[:2] + 'ay'
-
-    if re_capitlaize_word == True:
-         new_phrase += '{} '.format(pig_latin_word.capitalize() + new_punctuation )
-    else:
-            new_phrase += '{} '.format(pig_latin_word + new_punctuation)
-    new_punctuation = ''
-    i +=1
-
-# 4. Print
-
-print(new_phrase)
\ No newline at end of file
+import string
+
+# 1. Setup Define/Functions
+
+
+FIRST_VOWELS = 'a e i o u A E I O U'.split()
+LATER_VOWELS = 'a e i o u y A E I O U Y'.split()
+SPECIAL_CASES = 'Ybe ybe Ycleped ycleped Yclept yclept Ysgard Ydo ydo Ydrad ydrad Yfere yfere Ygdrasyl ygdrasyl Yghe'\
+               'ygh Ygo ygo Yground yground Ypres ypres Ypsiliform ypsiliform Ypsiloid ypsiloid Yren yren Ysame ysame '\
+               'Ytterbic ytterbic Ytterbium ytterbium Yttria yttria Yttric yttric Yttriferous yttriferous Yttrious ' \
+               'yttrious Yttrium yttrium Yttro-cerite yttro-cerite Yttro-columbite yttro-columbite Yttro-tantalite ' \
+               'yttro-tantalite Yvel yvel Ywar ywar Ywis ywis'.split()
+PUNCTUATIONS = list(string.pu
+nctuation)
+CAPITALS = list(string.ascii_uppercase)
+
+
+# 2. Define Functions
+
+
+def get_input():
+    """Aquires english phrase from user input and casts it as a list of strings.  """
+    typed_phrase = input("What would you like to Anslatetray? ").split()
+    return typed_phrase
+
+
+def get_capital_indices(words):
+    """Aquires indices of words with capital letters in the first part of the word for recapitalization
+    >>> get_capital_indices(['I','like','to','eat','Bananas', 'and', 'mcDonalds'])
+    [0,4]
+    """
+    #for loop? List comprehension?
+
+
+def make_initial_letters_lowercase(words):
+    """Sets first letter in each word in words to be lowercase.
+    >>> make_initial_letters_lowercase(['I','like','to','eat','Bananas', 'MackleMore', 'mcDonalds'])
+    ['i', 'like', 'to', 'eat', 'bananas', 'mackleMore', 'mcDonalds']
+    """
+    # for loop? List comprehension?
+
+def translate_to_pig_latin(words):
+    """Translates each word in words to pig latin
+    >>> translate_to_pig_latin(['i','like','to','eat','bananas,', 'mackleMore', 'mcDonalds.'])
+    ['iyay','ikelay','otay','eatyay','ananasbay,', 'ackleMoreay', 'onaldsmcDay']
+    """
+
+    for word in words, latin_func(word) = ???
```

Comment:
This code looks incomplete? Did we forget some commits in your branching fix up? Let me know.